agent/src/agent/pipeline/config/stages/source/influx2.py

```diff
@@ -0,0 +1,30 @@
+from urllib.parse import urljoin
+from agent.pipeline.config.stages.influx import InfluxScript
+
+
+class Influx2Source(InfluxScript):
+    JYTHON_SCRIPT = 'influx2.py'
+
+    def get_config(self) -> dict:
+        config = super().get_config()
+        config['scriptConf.params'].extend([
+            {'key': 'URL', 'value': self._get_url()},
+            {'key': 'HEADERS', 'value': self._get_headers()},
+            {'key': 'DATA', 'value': self._get_data()},
```

Comment:
1. I would call it query instead of data
2. What about filtering conditions?
3. Maybe it's better to allow the user to write his own queries as we do for victoria agents etc.?